src/App.tsx

```diff
@@ -9,34 +10,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is&nbsp;Tab 1
-      </h1>
-
-      <div>
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active">
-              <a href="#tab-1">Tab 1</a>
-            </li>
+  const [selectedTabId, setTab] = useState(tabs[0].id);
 
-            <li className="">
-              <a href="#tab-2">Tab 2</a>
-            </li>
+  const onClickTabItem = (id: string) => {
+    const tab = tabs.find(el => el.id === id);
```

Comment:
Pay attention to variable names, what is el?

Reply:
And will be better to change selected tab id only if new tab id is not equal to current tab id